### ISSUE TITLE
Can't search in multiple folders

### ISSUE THREAD

**Issue by Korne127** [eyalroz/removedupes#239]
I'm using Thunderbird 128.5.0esr on macOS 14.3.
When I right-click a single folder, the context menu contains the two relevant options of this add on (the lowest two):
<img width="210" alt="image" src="https://github.com/user-attachments/assets/47fec8c4-d18c-4139-8107-ae6d0580fa33">

However, when I select several folders and right-click, the context menu does not contain them.
<img width="214" alt="image" src="https://github.com/user-attachments/assets/631ef284-69f5-4e3d-8655-738202e1bfdc">

Furthermore, after you right-clicked on a selection of several folders just a single time, the two relevant options won't appear at all anymore until you restart Thunderbird:
<img width="212" alt="image" src="https://github.com/user-attachments/assets/70165e7c-f103-4756-b688-9f5d093d8db5">

**Comment by eyalroz** [comment]
That's true, Thunderbird 128 seems to have introduced a context menu with a different designation for multiple folders. I need to figure out how to overlay that one as well as the original one.

**Comment by tai8pan** [comment]
Hi, the problem is not the context menu. The actions can be found in Tools menu.
But the problem is that they do not work if multiple folders are selected. It says then "No folders selected" and abort operation.

Is there a chance you will fix it. 

Without ability to search across the folders the usability of the plugin is very limited.

**Comment by eyalroz** [comment]
> Is there a chance you will fix it.

Yes. But - I am in the middle of resisting a war right now - the genocidal campaign against Gaza, as well as the assaults on Syria and Lebanon. My free time is almost entirely taken up by trying to organize protest activities. I'll try to squeeze this in, but no promises for the next few months at least.

**Comment by tai8pan** [comment]
Oh, sorry. I didn't realize your situation. 
Totally understand that you don't have other priorities right now.

I wish you that nightmare will finish soon and you, your family and friends can rest safe.

**Comment by eyalroz** [comment]
I can replicate the bug. There is either one or two problems.

One potential-problem, is that my overlay regards the outer window, the 3-pane message window, while the folderPaneContext element itself is in the inner window, `main3PaneTabBrowser1`. That's true both with a single and with multiple folders selected though, so it might just be an awkward but working placement .

The second problem is that some of the items on the context menu disappear when multiple folders are selected - and I don't know how they are made to disappear, nor what decides which items disappear. Frustrating.

**Comment by tai8pan** [comment]
Hi,

Yes, the options in Tools.. menu are visible, but they do not work when multiple folders are selected.
I can click them but nothing happens ☹️

**Comment by eyalroz** [comment]
Yes, that's true. I'll look into it. Damn it, they  pull the rug from under our feet without even telling us what they've done :-(

**Comment by eyalroz** [comment]
Please try this build:

[removedupes_0.6.2b7_tbird.zip](https://github.com/user-attachments/files/19595629/removedupes_0.6.2b7_tbird.zip)

Multi-folder search should work now (again, via the menus; the context menu item visibility is a different issue)

**Comment by eyalroz** [comment]
This should be fixed by now (v0.6.6); please reopen if the problem persists.
